src/bindings/glib/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ else()
 	target_compile_options (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_CFLAGS_OTHER})
 
 	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC elektra-core elektra-kdb)
-	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_LDFLAGS})
+	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_LIBRARIES})
```

Comment:
@manuelm what do you think here? Is this a bug?

A separate PR (or at least commit) for this might be useful (if this change should be done)